jbmc/unit/Makefile

```
@@ -144,9 +144,9 @@ all: $(CATCH_TEST)
 clean: java-testing-utils-clean
 
 test: $(CATCH_TEST)
-	if ! ./$(CATCH_TEST) -l | grep -q "^$(N_CATCH_TESTS) test cases" ; then \
-	  ./$(CATCH_TEST) -l ; fi
-	./$(CATCH_TEST)
+	if ! ./$(CATCH_TEST) *,[.] -l | grep -q "^$(N_CATCH_TESTS) matching test cases" ; then \
```

@hannes-steffenhagen-diffblue how about?
```suggestion
  # Include hidden tests by specifying "*,[.]" for tests to count
	if ! ./$(CATCH_TEST) *,[.] -l | grep -q "^$(N_CATCH_TESTS) matching test cases" ; then \
```

---

Oh. Ok, I thought this was using a glob pattern somehow. Would you mind putting quotes around it to clarify it’s supposed to be passed as an argument to catch as-is?

---

Done - will merge on green CI